tekton/pipelines/binary-container.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: binary-container-0-1  # dot is not allowed in the name
+spec:
+  params:
+    - name: OSBS_IMAGE
+      description: The location of the OSBS builder image (FQDN pullspec)
+      type: string
+    - name: USER_PARAMS
+      type: string
+      description: User parameters in JSON format
+
+  workspaces:
+    - name: ws-build-dir
+    - name: ws-context-dir
+    - name: ws-registries-secret
+    - name: ws-koji-secret
+    - name: ws-worker-tokens-secrets
+    - name: ws-reactor-config-map
+    - name: ws-reactor-config-map-scratch
+
+  tasks:
+    - name: prebuild-binary-container
+      taskRef:
+        name: prebuild-binary-container-0-1
+      workspaces:
+        - name: ws-build-dir
+          workspace: ws-build-dir
+        - name: ws-context-dir
+          workspace: ws-context-dir
+        - name: ws-registries-secret
+          workspace: ws-registries-secret
+        - name: ws-koji-secret
+          workspace: ws-koji-secret
+        - name: ws-worker-tokens-secrets
+          workspace: ws-worker-tokens-secrets
+        - name: ws-reactor-config-map
+          workspace: ws-reactor-config-map
+        - name: ws-reactor-config-map-scratch
+          workspace: ws-reactor-config-map-scratch
+      params:
+        - name: OSBS_IMAGE
+          value: "$(params.OSBS_IMAGE)"
+        - name: USER_PARAMS
+          value: "$(params.USER_PARAMS)"
+    - name: build-binary-container
+      taskRef:
+        name: build-binary-container-0-1
+      workspaces:
+        - name: ws-build-dir
+          workspace: ws-build-dir
+        - name: ws-context-dir
+          workspace: ws-context-dir
+        - name: ws-registries-secret
+          workspace: ws-registries-secret
+        - name: ws-koji-secret
+          workspace: ws-koji-secret
+        - name: ws-worker-tokens-secrets
+          workspace: ws-worker-tokens-secrets
+        - name: ws-reactor-config-map
+          workspace: ws-reactor-config-map
+        - name: ws-reactor-config-map-scratch
```

Comment:
remove the scratch reactor config map

we are using just 1 reactor config, and setting it to correct one in osbs-client

Reply:
removed it.